src/toast/redux/toast.selectors.js

```diff
@@ -0,0 +1 @@
+export const getCurrentToast = state => state.ui.toast[0]
```

Comment:
You never can stack toasts ?

Reply:
No, only one toast is displayed at a time. 
https://material.io/guidelines/components/snackbars-toasts.html#snackbars-toasts-usage